src/tests/conformance/query/get_by_hash.rs

```diff
@@ -74,3 +75,49 @@ async fn should_get_transaction_by_hash() {
     assert!(synth_node.expect_message(&check).await);
     node.stop().expect("unable to stop stateful node");
 }
+
+#[tokio::test]
+async fn should_query_for_transactions_after_have_transactions() {
+    // ZG-CONFORMANCE-009
+    // Create stateful node.
+    let target = TempDir::new().expect("unable to create TempDir");
+    let mut node = build_stateful_builder(target.path().to_path_buf())
+        .expect("unable to get stateful builder")
+        .start(false)
```

Comment:
unrelated to this PR, but shouldn't we specify logging at the builder stage?

Reply:
It feels "closer" to what it does here, but I'm not insisting. Added TODO.